### ISSUE TITLE
InvalidBlockTag: @ symbol used in an @code block

### ISSUE THREAD

**Issue by ben-manes** [google/error-prone#1673]
### Description of the problem / feature request:

A `@code` block escapes special characters, like `@` for annotations.

```
 * {@code
 *   @GET @Path("/users/{id: " + PathPatterns.UUID + "}")
 * }
```

### Feature requests: what underlying problem are you trying to solve with this feature?
The following is a false positive.

```
warning: [InvalidBlockTag] Tag name `GET` is unknown. Did you mean tag `see`?
 *   @GET @Path("/users/{id: " + PathPatterns.UUID + "}")
     ^
    (see https://errorprone.info/bugpattern/InvalidBlockTag)
  Did you mean '*   @see @Path("/users/{id: " + PathPatterns.UUID + "}")'?
```

**Comment by cushon** [comment]
This is an issue with how OpenJDK parses javadoc, not with Error Prone. Prior to JDK 15, that doesn't work if the `{@code ...}` block contains newlines: https://bugs.openjdk.java.net/browse/JDK-8241780

**Comment by ben-manes** [comment]
Oh, thanks! I'll fix the newline instead then.
